SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialWorldSettings.cpp

```diff
@@ -5,9 +5,12 @@
 #include "SpatialGDKSettings.h"
 #include "Utils/SpatialDebugger.h"
 
+#if WITH_EDITOR
+bool ASpatialWorldSettings::bDisableMultiWorker = false;
```

Comment:
I'm not sure we need this variable - can't we just access the GDKEditorSettings::bDisableMultiWorker directly?

Reply:
I tried but I was unable to include SpatialGDKEditorSettings.h in WorldSettings.cpp.

Reply:
Discussed offline, we'll put this in the SpatialGDKSettings instead, but only use it when running in editor.

Reply:
Moved to SpatialGDKSettings.